python/ray/tests/test_dashboard.py

```diff
@@ -195,16 +194,38 @@ def f():
 def test_dashboard_agent_grpc_port_conflict(listen_port, call_ray_start):
     address = call_ray_start
     ray.init(address=address)
+
     # Tasks without runtime env still work when dashboard agent grpc port conflicts.
     run_tasks_without_runtime_env()
     # Tasks with runtime env couldn't work.
     with pytest.raises(
         ray.exceptions.RuntimeEnvSetupError,
-        match="the grpc service of agent is invalid",
+        match="Ray agent couldn't be started due to the port conflict",
```

Comment:
I updated this test because it seemed to be failing independently of my changes.
(I can't find anything in the code base that matches the original regex match)

I can see it is skipped on Windows, does it usually get run in CI pipelines?